src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(0);
```

Comment:
1) 0 - is not a tabId. It's index
2) it would be better if you add the tab to the state (like tabs[0])